File: dayfour/Palindrome.java
public class Palindrome{
	public static Boolean[] IsPalindrome(){
		String [] list={"madam", "kali", "hello", "noon"};
		Boolean[] Array = new Boolean[list.length];
			for(int count = 0; count < list.length; count++){
				String palindrome = "";

				for(int counter = 0; counter < list[count].length(); counter++){
					palindrome = list[count].charAt(counter) + palindrome;
				}

				if (list[count].equals (palindrome))
					Array[count] = true;
				else
					Array[count] = false;
			}
			return Array;
		
		}








}

File: dayfour/palindrome.py
def element_is_palindrome():
	my_element  = ["madam", "kali", "hello", "noon"]
	count = 0
	for elements in my_element:
		pals = ""
		for element in elements:
			pals = element + pals
		if elements == pals:
			my_element[count] = True;
		else:
			my_element[count] = False;
		count+=1
	return my_element
print(element_is_palindrome())